tools/packaging/debian/create_deb.sh

```diff
@@ -46,15 +46,15 @@ cp LICENSE "${AOC_ROOT}/opt/aws/aws-otel-collector/"
 cp VERSION "${AOC_ROOT}/opt/aws/aws-otel-collector/bin/"
 cp "build/linux/${ARCH}/aoc" "${AOC_ROOT}/opt/aws/aws-otel-collector/bin/aws-otel-collector"
 cp tools/ctl/linux/aws-otel-collector-ctl.sh "${AOC_ROOT}/opt/aws/aws-otel-collector/bin/aws-otel-collector-ctl"
-cp config.yaml "${AOC_ROOT}/opt/aws/aws-otel-collector/etc"
+# default configuration
```

Comment:
adding this to this pr since this is a one liner.

This change will make the .deb file in parity with the RPM file and how the control script works.